modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -4,13 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-With Anypoint Exchange's https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[Maven Facade], Maven clients can publish and consume these Exchange assets:
+With Anypoint Exchange's https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[Maven Facade], Maven clients can publish and consume Exchange assets, including these asset types:
```

Comment:
GLOBALLY avoid making trademarks and other proprietary names possessive, per style ref and term pres list. You can just use the plain form, which acts as an adjective or nonhyphenated compound adjective.

```suggestion
With Anypoint Exchange https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[Maven Facade], Apache Maven clients can publish and consume Exchange assets, including these asset types:
```

Reply:
Adde "API" after Maven Facade?